platform/marvell/sonic-platform-nokia/debian/sonic-platform-nokia-7215.postinst

```diff
@@ -9,6 +9,8 @@ systemctl restart kmod
 systemctl enable nokia-7215init.service
 systemctl start nokia-7215init.service
 
+systemctl mask watchdog-control.service
```

Comment:
Is this guaranteed to happen before `watchdog-control.service` starts? If there's a chance it could be after, then add `--now` to have systemd stop the service as well.

Reply:
I have verified all scenarios to make sure there is no race condition - fresh install, software upgrade, reboots. The `watchdog-control.service` is always masked correctly and the platform watchdog is active.

I have added the --now flag as recommended. I agree this is good to have in case something changes in the future